### PR TITLE
Add Newsletter 130-131

### PR DESCRIPTION
**Pull request by bthrall**

### CODE REVIEW

**Comment by bthrall** [comment]
@jandem would you mind reviewing and posting a notice on Mastodon when the PR is merged?

File: _posts/2024-09-12-newsletter-firefox-130-131.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "SpiderMonkey Newsletter (Firefox 130-131)"
+date: 2024-09-12 15:33:08 -0500
+---
+
+# SpiderMonkey Newsletter 130-131
+
+Hello everyone\!
```
Comment (mgaudet) [inline _posts/2024-09-12-newsletter-firefox-130-131.markdown:9]: I think I am learning something about Google's markdown export... punctuation is dangerous and must be escaped.

Comment (bthrall) [reply _posts/2024-09-12-newsletter-firefox-130-131.markdown:9]: Yeah :disappointed:

**Comment by jandem** [comment]
Thank you, @bthrall! I skimmed [the preview](https://deploy-preview-183--spidermonkey-dev.netlify.app/blog/2024/09/12/newsletter-firefox-130-131) and it all looks good to me.

**Comment by bthrall** [comment]
I don't yet have the ability to approve or merge this, so if it looks good, would you please do so?

**Review by jandem — APPROVED** [review]

**Comment by jandem** [comment]
Done! Also posted here: https://mozilla.social/@SpiderMonkey/113130765678869526

**Pull request merged**